api/query_engine.go

```diff
@@ -8,9 +8,7 @@ import (
 )
 
 var (
-	numPointsFetch  = stats.NewMeter32("api.align_requests.num_points_fetching", false)
-	numPointsReturn = stats.NewMeter32("api.align_requests.num_points_returning", false)
-	chosenArchive   = stats.NewMeter32("api.align_requests.chosen_archive", false)
+	chosenArchive = stats.NewMeter32("api.align_requests.chosen_archive", false)
```

Comment:
chosenArchive should be with the other 2. both in terms of where they are defined (here or in api.go I don't really care, though I think here is the better place for all 3 since the variables are used in this file. they are documented together anyway so it's not like they have to be in the code together)

and also in terms of the namespace, so this one should be `api.request.render.chosen_archive`